python/paddle/distributed/utils/log_utils.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+
+def get_logger(log_level, name="root"):
```

Comment:
why we need another 'get_logger'? there are multiple versions of this tiny utility.
$ grep -irn 'def get_logger' python
python/paddle/distributed/utils/log_utils.py:18:def get_logger(log_level, name="root"):
python/paddle/distributed/auto_parallel/utils.py:31:def get_logger(log_level, name="auto_parallel"):
python/paddle/distributed/fleet/utils/http_server.py:30:def get_logger(name, level, fmt):
python/paddle/distributed/fleet/launch_utils.py:267:def get_logger(log_level=20, name="root"):
python/paddle/fluid/tests/unittests/auto_checkpoint_utils.py:44:def get_logger():
python/paddle/fluid/incubate/fleet/utils/http_server.py:25:def get_logger(name, level, fmt):
python/paddle/fluid/log_helper.py:22:def get_logger(name, level, fmt=None):

Reply:
This should be resolved later.